### ISSUE TITLE
Add readme in other languages

### ISSUE THREAD

**Issue by Bost** [opencovid19-fr/data#141]
While discussing about "how to scale the data gathering done by CSSEGISandData/COVID-19" I was [notified](https://github.com/CSSEGISandData/COVID-19/issues/1035#issuecomment-601359232) about your consolidation effort. So I followed the reference and... and I see everything is in your national language. Oh non. Pourquoi?!?

Well, using national language(s) may make the work easier for you but it effectively seals off any communication between yours and other [similar](https://github.com/pcm-dpc/COVID-19) project(s) or in the [better case ](https://coronadatascraper.com/#sources) makes if one-directional.

Nobody from the rest of the world can learn from your example, nobody else can contribute to your effort. Everyone has to start his/her own effort from the scratch, trip over the same mistakes, etc.

Can you do something about it guys?

We have a global problem I doubt we can solve it every nation on its own.

**Comment by payoto** [comment]
Hey @Bost, 

the issue is balancing the effectiveness of our effort to consolidate data from French gov and hospital sources (all in French) and the broader use to the international community. We would be unduly restricting our contributor pool by requiring both French and English language skills, and right now we need every hand we can get to fill the gaps in the datasets.

So at this stage it makes sense that the docs and data sets be in French as the bulk of the contributors will be from France.

This is also a young project and we are currently consolidating our organisation in the background, once things start running smoother (hopefully in the next days) international access will definitely be something to consider!

Anyway, hope you stick around for when we get this sorted. Are you part of another similar project?

**Comment by Bost** [comment]
> Are you part of another similar project?

I live on the same planet, so yes. Sort of :) BTW I speak some 4 languages. One of it yours (B2 - C1 level). If that helps...

**Comment by aurelienpierre** [comment]
Hey, I can help translating stuff FR <-> EN. I'm new to this, so I will need some guidance to start (but I use Poedit/Python/Git is no issue).

**Comment by Bost** [comment]
> Hey, I can help translating stuff FR <-> EN. I'm new to this, so I will need some guidance to start (but I use Poedit/Python/Git is no issue).

I already sent a PR with translation of the README.md. https://github.com/opencovid19-fr/data/pull/149 please search for any other FR documents, and if you find any, translate it.

**Comment by ColinMaudry** [comment]
I will add to what @payoto commented. The authorities of each country have their way to publish data. This project is tailored to deal with how French authorities publish data:

- temporality
- variety of sources
- lack of structured data

One thing we could do is make the result of we do (data and code) easier to read for foreigners:

- translate data key names (`casConfirmes` => `confirmedCase`)
- document how we gathered and compile data

In a nutshell: I also believe it would be counterproductive to switch communication to English, but can find a way to facilitate the accessibility of the output. And of course, questions in English in issues are welcome :)

**Comment by Binnette** [comment]
The readme is now available in english.
Please feel free to make pull requests to improve it :wink:
